package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mongoose-crud",
+  "version": "0.0.1",
+  "description": "Express middleware that creates all the CRUD endpoints for a mongoose model.",
+  "main": "lib/crud.js",
+  "scripts": {
+    "build": "babel --presets es2015 crud.js --out-dir lib --ignore test.js"
+  },
+  "keywords": [
+    "MongooseJS",
+    "MongoDB",
+    "Express",
+    "CRUD"
+  ],
+  "author": "Weston <Weston@blazingsolutions.net>",
+  "license": "MIT",
```

Comment:
Add a LICENSE file (GH provides a nice way to do this, let me know if you want me to show you)